.github/ISSUE_TEMPLATE/work-item-issue.md

```diff
@@ -0,0 +1,81 @@
+# Work Item Issue Template
+
+## 1. **Title** *
+Provide a concise and descriptive title for the work item.
+
+Your input ...
+
+## 2. **Problem Statement** *
+Describe the problem, issue, or opportunity that this work item addresses.
+- **What is the problem?**: Explain the context and why it needs resolution.
+- **Impact**: Describe how the problem affects users, systems, or the project.
+
+Your input ...
+
+## 3. **Objective** *
+State the objective of this work item. What outcome is expected?
+- **Success Criteria**: Define how success will be measured (e.g., KPIs, metrics).
+
+Your input ...
+
+## 4. **Supporting Documentation** *
+Include links to supporting documents such as:
+- Data Examples: [Link to data or relevant files]
+- Related Use Cases or Discussion Documents: [Link to discussion]
+- PRs or Other References: [Link to relevant references]
+
+Your input ...
+
+## 5. **Proposed Solution / Approach**
+Outline any proposed solutions, approaches, or potential paths forward.
+- **Initial Ideas**: Describe potential solution paths, tools, or technologies.
+- **Considerations**: Include any constraints, dependencies, or risks.
+
+Your input ...
+
+## 6. **Epic or Theme Association**
+Indicate if this issue is part of a broader epic or theme. _(OPTIONAL)_
+- **Epic**: [Epic Name or Number]
+- **Theme**: [Theme Name, if applicable]
+
+## 7. **Stakeholders** *
+List the main stakeholders for this issue.
+- **Primary Stakeholder**: [Name/Role]
+- **Other Involved Parties**: [Names/Roles]
+
+## 8. **Priority and Effort Estimation**
+Provide a rough estimation of the effort required and the priority level.
+- **Effort Level**: Low / Medium / High
+- **Priority**: P1 / P2 / P3 (define based on project standards)
```

Comment:
It's too soon to decide priority and level of effort at this stage.  This should be part of triage and stakeholder input.  I want the work item creation process to be laser-focused on declaration of the problem, not an assessment of importance and effort.  Propose striking.

Reply:
Same here. I see it as a placeholder for the Maintainers to add these details later during the elaboration step. We could add a comment to discourage completion at an early stage.

Reply:
@shawnalpay See later changes. I have added a comment indicating that the Maintainers will complete the section. What do you think?